src/emptyState/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import cc from 'classcat'
+import Title from 'title'
+import style from './style'
+
+interface EmptyState {
+  readonly className?: Classcat.Class
+  readonly image: string
+  readonly text: string
+  readonly button?: JSX.Element
+}
+
+const EmptyState = ({ className, image, text, button}: EmptyState) => (
+  <div className={cc(['kirk-empty-state', className])}>
+    <img src={image} alt="" />
+    <Title>{text}</Title>
```

Comment:
I would put `text` prop as a `children`, that's what we normally do in our component: main test is the `children` prop, wdyt?

Reply:
I thought about that part a bit: children is too permissive with what we can give it, allowing nonsenses in the `Title` component.
But... as the button can be whatever we want, i'll be more open to use it as children.

Reply:
Ok my bad, that's not what we do anymore in components apparently, checked with Item(s) components.